src/backend/marsha/core/tests/test_models_passport.py

```diff
@@ -26,27 +31,58 @@ def test_models_lti_passport_str(self):
         pl_passport.delete()
         self.assertEqual(str(pl_passport), "clé_pl [pl][deleted]")
 
-    def test_models_lti_passport_fields_nothing_required(self):
-        """The LTIPassport model can be instantiated without setting any field.
+    def test_models_lti_passport_fields_playlist_or_consumer_site_required(self):
+        """Creating an LTIPassport requires either a consumer site or a playlist."""
```

Comment:
`s/an LTIPassport/a LTIPassport/` ?

If you agree with this you can make this replacement everywhere in this file.